resources/assets/components/InboxItem/InboxTile.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class InboxTile extends React.Component {
+  displayImage(photo_url) {
```

Comment:
@sbsmith86 I'm not exactly sure what you mean by "take in a property," I thought that's what it was doing?